### DISCUSSION TITLE
Gist Embeds Broken?

### DISCUSSION THREAD

**Discussion by sparkweb** [community/community#24715] [Apps, API and Webhooks]
<p>All of our Gist embeds seem to be broken this morning. It looks like all the content is being HTML encoded. Open this up directly in a web browser and you can see everything is being HTML encoded.</p>
<p><code>https://gist.github.com/sparkweb/9954610036d69293a19c4eadffd022aa.js</code></p>
<p>Here’s how it shows on my site: <a href="https://help.orderdesk.com/article/612-how-to-create-a-text-or-xml-file#template" rel="noopener nofollow ugc">https://help.orderdesk.com/article/612-how-to-create-a-text-or-xml-file#template</a> (Sample XML Template section).</p>

**Comment by ElvinD** [comment]
<p>You were just ahead of me. I came here because of all the Medium Gist embeds. They all seem to be broken, showing HTML markup instead of rendered content:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/8/d88c306e089de49a7831efd397c2f95edd3c2e5c.png" data-download-href="/uploads/short-url/uTFue8hgWE4FQg4Q4f4AigIzvMo.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/7555096/181086416-13333154-fc9b-4c20-96fb-bf80785a6855.png" alt="image" data-base62-sha1="uTFue8hgWE4FQg4Q4f4AigIzvMo" width="690" height="228" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/8/d88c306e089de49a7831efd397c2f95edd3c2e5c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">713×236 14.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by MC-Guenette** [comment]
<p>Also seeing this issue! :frowning: Tried looking for a solution, but all embed scripts are broken</p>

**Comment by ghost** [comment]
<p>Also seeing this issue.</p>

**Comment by parzibyte** [comment]
<p>Same here :frowning:<br>
Tested in Chrome, Firefox and Edge, it’s not working (I tought it was due to document.write but I haven’t seen any breaking api changes). Any solution or workaround?</p>

**Comment by restored316** [comment]
<p>We are also experiencing the same thing on all blog posts and helpscout documents.</p>

**Comment by nethgato** [comment]
<p>Hey all :wave:</p>
<p>Similarly to <a href="https://github.community/t/github-blame-view-shows-html-markup/149818" class="inline-onebox">Github blame view shows html markup</a>, there is an issue that our dev team is actively working to resolve.</p>
<p>The two are indeed related and hopefully we’ll have a resolution shortly. :crossed_fingers:</p>

**Comment by janpreet** [comment]
<p>I’ve been able to embed gist in the past but something changed and I can no longer add gist’s to my site. It’s exposed tags now. For example (this is not my site, but I have the exact same issue)-<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <a href="https://medium.com/@aryamurali/embed-code-in-medium-e95b839cfdda" target="_blank" rel="noopener nofollow ugc" title="04:59AM - 01 June 2018">Medium – 1 Jun 18</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/430;"><img src="https://user-images.githubusercontent.com/6337111/181086427-19448955-d385-4532-8d24-45f996fd9420.png" class="thumbnail" width="690" height="430" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/b/ab51f70397ea15fc202f4e0da791c2797cb0a7e6_2_690x430.png, https://aws1.discourse-cdn.com/github/optimized/3X/a/b/ab51f70397ea15fc202f4e0da791c2797cb0a7e6_2_1035x645.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/a/b/ab51f70397ea15fc202f4e0da791c2797cb0a7e6.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/b/ab51f70397ea15fc202f4e0da791c2797cb0a7e6_2_10x10.png"></div>

<h3><a href="https://medium.com/@aryamurali/embed-code-in-medium-e95b839cfdda" target="_blank" rel="noopener nofollow ugc">Embed Code in Medium</a></h3>

<p>Embedding code in Medium is pretty easy. There are two ways to primarily do it — by embedding the code directly on to medium or embedding…</p>

  <p><span class="label1">Reading time: 2 min read</span>
    </p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
<br>
<div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/9/1989f84f9b3879b95c8ff19f782835ea380bc4e0.png" data-download-href="/uploads/short-url/3DVxez4ZOhLfdlURW3MssSeaf3q.png?dl=1" title="Screen Shot 2020-12-14 at 1.10.05 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/6337111/181086428-c1bdd0e8-3fc8-42ab-9667-5320198fad37.png" alt="Screen Shot 2020-12-14 at 1.10.05 PM" data-base62-sha1="3DVxez4ZOhLfdlURW3MssSeaf3q" width="646" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/9/1989f84f9b3879b95c8ff19f782835ea380bc4e0_2_646x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/1/9/1989f84f9b3879b95c8ff19f782835ea380bc4e0_2_969x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/1/9/1989f84f9b3879b95c8ff19f782835ea380bc4e0.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/9/1989f84f9b3879b95c8ff19f782835ea380bc4e0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-12-14 at 1.10.05 PM</span><span class="informations">1032×798 184 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by nethgato** [comment, marked as answer]
<p>Hey, all! :wave:</p>
<p>We should be all good.</p>
<p>Please let us know if that’s not the case for you! :bow:</p>

**Comment by sparkweb** [comment]
<p>Looks good to me. Thank you for the fix and the alert!</p>

**Comment by janpreet** [comment]
<p>Thank you for the fix!</p>

**Comment by parzibyte** [comment]
<p>It’s working now. Thank you very much :smiley:</p>

**Comment by ahandsel** [comment]
<p>Hello nethgato,</p>
<p>Sorry, we are still experiencing issues with our embedded Gists.</p>
<p>Like sparkweb’s website (first post), our website also embeds Gist into our tutorial pages to showcase code, mostly JS.</p>
<p>Currently, we are unable to select and copy the code in our embedded Gists. It was working perfectly fine last week but not this week.</p>
<p>Context:</p>
<ul>
<li>We host our content on Zendesk</li>
<li>Example page where we are experiencing the issue:<br>
<a href="https://developer.kintone.io/hc/en-us/articles/360000365282-Display-Maps-With-LeafletJS#locationIQ:~:text=Step%202.%20Create%20the%20leaflet%2Dmap.js%20JavaScript,below%20(replace%20the%20string%20Insert_Access_Token_Here%20)" rel="noopener nofollow ugc">https://developer.kintone.io/hc/en-us/articles/360000365282-Display-Maps-With-LeafletJS#locationIQ:~:text=Step%202.%20Create%20the%20leaflet-map.js%20JavaScript,below%20(replace%20the%20string%20Insert_Access_Token_Here%20)</a>
</li>
</ul>
<p>How might we go about fixing this issue?</p>
<p>Thank you for your time.</p>

**Comment by nethgato** [comment]
<p>Hi <a class="mention" href="/ahandsel">@ahandsel</a></p>
<p>Thanks for raising this. Though, when I load that example page, I’m not seeing any content fail to load. In my Chrome dev tools, I can see the src to your Gist and the content loads without issue.</p>
<p>Are there specific scenarios where the gist fails to load (specific browser? logged in/out?) or have you seen this more broadly?</p>
<p>For any remaining concerns on gists failing to load, they may require a new/unique thread. The OP and subsequent posts were related to a specific update that was rolled back, removing the embedded gist breaking changes.</p>
<p>Though perhaps I’m not understanding how to properly reproduce the problem.</p>

**Comment by nethgato** [comment]
<p>A post was split to a new topic: <a href="https://github.community/t/syntax-highlighting-on-embedded-gist-fails/151538">Syntax highlighting on embedded gist fails</a></p>

**Comment by CounselorAce** [comment]
<p>Same. I used to be able to share notebooks as gist from Jupyter. Now, I cannot.</p>
